### PR TITLE
Combining data from data/storysquad-ground-truth-31 and data/storysquad-ground-truth-51 into data/storysquad-ground-truth

### PR DESCRIPTION
**Pull request by cryptobellum**
Combined data from data/storysquad-ground-truth-31 and data/storysquad-ground-truth-51 into data/storysquad-ground-truth. Also, renamed files that had naming errors to eliminate issues when training the model.

The new merged data was used to train tesseract model. The tesseract model does not stop mid way like it was stopping earlier before the cleanup. 

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [X] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [X] Yes
- [ ] No
- [ ] Not necessary

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [X] There are no merge conflicts

[Loom Video](https://www.loom.com/share/a8273e77423442e2b1dfe32a841eb17f)

[Trello Card](https://trello.com/c/Su1l1RFK/656-combining-data-from-data-storysquad-ground-truth-31-and-data-storysquad-ground-truth-51-into-data-storysquad-ground-truth)

### CODE REVIEW

File: data/storysquad-ground-truth/51-5131-2-13.gt.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+to go ^ the shelter." Razzle knew
```
Comment (BritVandy) [inline data/storysquad-ground-truth/51-5131-2-13.gt.txt:1]: I can barely tell what words are in the clip. Good eye.

**Review by BritVandy — APPROVED** [review]
This looks like it was a lot of work. Good job guys!

**Comment by TashaSkyUp** [comment]
Looks good.

**Review by TashaSkyUp — APPROVED** [review]
Looks good.

**Review by peterger8y — APPROVED** [review]

**Pull request merged**